__tests__/app.test.js

```diff
@@ -92,6 +92,69 @@ describe("GET /api/reviews", () => {
         });
       });
   });
+  it("should return status 200, responds with review selected from category query value", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then((response) => {
+        const reviewsArray = response.body.reviews;
+        expect(reviewsArray.length).toBe(11);
```

Comment:
If we wanted to be a little extra robust here we could do a forEach and check that every element.category === 'social deduction'.